spec/api/v1/schemas/profiles.rb

```diff
@@ -57,6 +57,13 @@ module Profiles
               type: 'string',
               example: '0.1.49'
             },
+            values: {
+              type: 'object',
+              example: {
+                'uuid1': 'value1',
+                'uuid2': 'value2'
+              }
+            },
```

Comment:
We're storing `profile.value_overrides` as {value_ref_id: value} currently, not {uuid, value}
https://docs.google.com/document/d/1u9Yn4aYD3i0xrFNXbviJHRkXheyX5O0vCdYaDORkKT4/edit?usp=sharing

Reply:
We talked about it for `Rule#value_checks` where we decided to store that as `[uuid, uuid...]` and I mentioned that we used ref_ids for `Profile#value_overrides` and I think you said that we might need to change that in the future but we didn't decide on it yet

Reply:
Yes, we need it to be consistent

Reply:
Yeah, that makes sense. I can create a ticket to update it

Reply:
No need to create a ticket, I already pulled back the original one from testing to in progress. Just make a follow-up with the same JIRA number.

Reply:
Okay, thanks!